### PR TITLE
imageryintro.html: add section about semantic labels

### PR DESCRIPTION
**Pull request by neteler**
In the intro a section about semantic labels information is missing, this PR suggest a section.

It could be improved, edits are welcome.

### CODE REVIEW

File: imagery/imageryintro.html [hunk 1]
```diff
@@ -86,6 +86,19 @@ <h3>Imagery import</h3>
...
+Semantic labels are a description like band names for satellite imagery
+which can be stored as metadata. <a href="r.semantic.label.html">r.semantic.label</a>
+allows assigning a semantic label information to a single raster map
+or to a list of specified raster maps.
+To print available semantic labels used for multispectral data, use
+<a href="i.band.library.html">i.band.library</a>.
+Specific band reference can be assigned to a raster map as a semantic
+label by the modules <em><a href="r.semantic.label.html">r.semantic.label</a></em>
+or <em><a href="r.support.html">r.support</a></em>. Semantic labels
+are supported by <a href="temporalintro.html">temporal</a> GRASS modules.
```
Comment (marisn) [inline imagery/imageryintro.html:100]: and are used in signature files of imagery classification tools.

**Review by marisn — CHANGES_REQUESTED** [review]

File: imagery/imageryintro.html [hunk 2]
```diff
@@ -86,6 +86,19 @@ <h3>Imagery import</h3>
...
+
```
Comment (neteler) [inline imagery/imageryintro.html:101]:
```suggestion
are supported by <a href="temporalintro.html">temporal</a> GRASS modules
and are used in signature files of imagery classification tools.
```

File: imagery/imageryintro.html [hunk 3]
```diff
@@ -86,6 +86,19 @@ <h3>Imagery import</h3>
...
+Semantic labels are a description like band names for satellite imagery
+which can be stored as metadata. <a href="r.semantic.label.html">r.semantic.label</a>
+allows assigning a semantic label information to a single raster map
+or to a list of specified raster maps.
+To print available semantic labels used for multispectral data, use
+<a href="i.band.library.html">i.band.library</a>.
+Specific band reference can be assigned to a raster map as a semantic
+label by the modules <em><a href="r.semantic.label.html">r.semantic.label</a></em>
+or <em><a href="r.support.html">r.support</a></em>. Semantic labels
```
Comment (veroandreo) [inline imagery/imageryintro.html:99]: This sounds repetitive with the second sentence. AFAIU, r.semantic.label allows only to add the satellite imagery band references defined in i.band.library (see the Notes section in https://grass.osgeo.org/grass80/manuals/r.semantic.label.html), while r.support allows to add any sort of semantic label the user wishes. 

Maybe the info and figure I added in [New Features page](https://trac.osgeo.org/grass/wiki/Grass8/NewFeatures80) could be useful here.

Comment (veroandreo) [reply imagery/imageryintro.html:99]: There's a figure already in raster3dintro (https://grass.osgeo.org/grass78/manuals/raster3dintro.html). Is it complicated to store figs for intros? In my opinion, figures are helpful to explain concepts.

File: imagery/imageryintro.html [hunk 4]
```diff
@@ -86,6 +86,19 @@ <h3>Imagery import</h3>
...
+are supported by <a href="temporalintro.html">temporal</a> GRASS modules
+and are used in signature files of imagery classification tools.
 <h3>Image processing operations</h3>
```
Comment (neteler) [inline imagery/imageryintro.html:102]:
```suggestion
<h3>Semantic label information</h3>

Semantic labels are a description which can be stored as metadata.
To print available semantic labels relevant for multispectral satellite
data, use <a href="i.band.library.html">i.band.library</a>.
<a href="r.semantic.label.html">r.semantic.label</a> allows assigning of
these satellite imagery band references as defined in
<a href="i.band.library.html">i.band.library</a>. Semantic labels are
also used in signature files of imagery classification tools.

With <a href="r.support.html">r.support</a></em> any sort of semantic label
the user wishes may be added. Semantic labels are supported also by the
<a href="temporalintro.html">temporal</a> GRASS modules.

<h3>Image processing operations</h3>
```

Comment (neteler) [reply imagery/imageryintro.html:102]: Thanks for the suggestion: new attempt inserted.

File: imagery/imageryintro.html [hunk 5]
```diff
@@ -86,6 +86,20 @@ <h3>Imagery import</h3>
...
 
+<h3>Semantic label information</h3>
+
+Semantic labels are a description which can be stored as metadata.
+To print available semantic labels relevant for multispectral satellite
+data, use <a href="i.band.library.html">i.band.library</a>.
+<a href="r.semantic.label.html">r.semantic.label</a> allows assigning of
+these satellite imagery band references as defined in
+<a href="i.band.library.html">i.band.library</a>. Semantic labels are
+also used in signature files of imagery classification tools.
```
Comment (veroandreo) [inline imagery/imageryintro.html:97]:
```suggestion
also used in signature files of imagery classification tools. Therefore, 
signature files of one imagery or raster group can be used to classify
a different group with identical semantic labels.
```

File: imagery/imageryintro.html (hunk 3 above)
Comment (neteler) [reply imagery/imageryintro.html]: Most things are doable :) See 914ff2c671ba7b0f007fda934c2a5ee79b0ee2b4

Remaining issue: I don't understand how the copying of `lib/raster3d/raster3d_layout.png` works to replicate it for `lib/imagery/band_references_scheme.png`? Where is the Makefile magic hidden?

File: imagery/imageryintro.html [hunk 6]
```diff
@@ -86,6 +86,20 @@ <h3>Imagery import</h3>
...
+
+With <a href="r.support.html">r.support</a></em> any sort of semantic label
+the user wishes may be added. Semantic labels are supported also by the
```
Comment (veroandreo) [inline imagery/imageryintro.html:107]:
```suggestion
the user wishes may be added (i.e., not only those registered in 
<em>i.band.library</em>). Semantic labels are supported also by the
```

**Review by veroandreo — COMMENTED** [review]
I added minor clarification, what do you think?

**Comment by neteler** [comment]
> I added minor clarification, what do you think?

Thanks!

TODO: Now only the copying of the PNG file is missing in the Makefile (or wherever than happens).
Does anyone have a hint?

File: imagery/imageryintro.html (hunk 3 above)
Comment (neteler) [reply imagery/imageryintro.html]: OK, got it, that image is installed by a raster3d module and just happens to be in the right place.

**Comment by neteler** [comment]
> > I added minor clarification, what do you think?
> 
> Thanks!
> 
> TODO: Now only the copying of the PNG file is missing in the Makefile (or wherever than happens). Does anyone have a hint?

Solved in 0613620. From my side, this PR should now be complete.

**Comment by neteler** [comment]
It now looks like this:

image

**Review by veroandreo — APPROVED** [review]

**Comment by neteler** [comment]
I take liberty to merge this, for GRASS GIS 8.0.0RC2.
Thanks for your feedback!

**Pull request merged**